lib/seahorn/BvOpSem2Context.hh

```diff
@@ -329,18 +329,20 @@ public:
   virtual Expr doTrunc(Expr op, unsigned bitWidth) = 0;
   virtual Expr doZext(Expr op, unsigned bitWidth, unsigned opBitWidth) = 0;
   virtual Expr doSext(Expr op, unsigned bitWidth, unsigned opBitWidth) = 0;
-  virtual Expr Extract(Expr op, unsigned begin, unsigned end) = 0;
-  virtual Expr Concat(Expr opHigh, Expr opLow) = 0;
+  virtual Expr Extract(Expr op, unsigned begin, unsigned end,
```

Comment:
how about changing `Extract` and `Concat` to take a pair `std::pair<Expr, unsigned>` to make sure that it is clear that bitwidth comes with the argument. And in code, use `{e, 5}` type of C++11 notation.

Reply:
Done